v1/api/swagger/components/schemas/Allocation.yaml

```diff
@@ -0,0 +1,26 @@
+type: object
+description: allocation of servers or storage for each PCE service
+properties:
+  ResourceType:
+    type: string
+    description: Type of the resource the allocation inforamtion is listed. It is either Server or Volume
+  Total:
+    type: integer
+    description: If the resource type is Server, Total is total number of servers.  If the resource type is Volume, Total  is the total storage capacity in TB
+  InstanceFamily:
+    type: string
+    description: Instance type family name.
+    example: Compute Optimized, General Purpose
```

Comment:
Are these possible choices? If choices, it would be good to list them in the 'description'.

Reply:
they are just examples. Not possible choices